src/web/src/pages/opportunities/[[...query]]/index.tsx

```diff
@@ -575,36 +591,36 @@ const Opportunities: NextPageWithLayout<InputProps> = ({
 
       <div className="container z-10 max-w-7xl px-2 py-1 md:py-4">
         <div className="flex flex-col gap-2 pb-2 pt-8 text-white">
-          <h3 className="flex flex-grow flex-wrap items-center justify-center">
+          <h3 className="-mb-1 flex flex-grow flex-wrap items-center justify-center text-xl font-semibold">
             Find <span className="mx-2 text-orange">opportunities</span> to
             <span className="mx-2 text-orange">unlock</span> your future.
           </h3>
-          <h5 className="text-center">
+          <h6 className="text-center text-[14px] font-normal text-[#C3A2CD]">
             A learning opportunity is a self-paced online course that you can
             finish at your convenience.
-          </h5>
+          </h6>
           <div className="my-4 md:items-center md:justify-center">
             <div className="flex flex-row items-center justify-center gap-2">
               <SearchInputLarge
                 onSearch={onSearchInputSubmit}
                 placeholder="What are you looking for today?"
                 defaultValue={query as string}
               />
-              <button
+              {/* <button
```

Comment:
why was the button removed?  I know it wasn't in the design, but because there's limited space available on the full-screen view (can't show all the categories etc), i specifically put the button there so that users can still open the filter dialog and see all the categories etc...